### ISSUE TITLE
[Bug]: Placeholders not working properly

### ISSUE THREAD

**Issue by jgibbz328** [EncryptSL/LiteEco#30]
### What happened?

I recently updated to the newest snapshot 1.2.3 from 1.2.0 and the placeholders do not seem to be working anymore with my holographic leaderboard.  I deleted the old config.yml and translations as stated before restarting the server, but it doesnt seem to have helped.  The leaderboard was working prior to the update.

### Plugin Version

1.2.3-SNAPSHOT

### You detected problem on this server platform ?

PaperMC

### Relevant log output

```shell
26.01 00:20:42 [Server] INFO Loading internal permission managers...
26.01 00:20:42 [Server] INFO Performing initial data load...
26.01 00:20:43 [Server] INFO Successfully enabled. (took 2449ms)
26.01 00:20:43 [Server] INFO Enabling Vault v1.7.3-b131
26.01 00:20:43 [Server] WARN Loaded class com.earth2me.essentials.api.Economy from Essentials v2.20.0-dev+39-312d169 which is not a depend or softdepend of this plugin.
26.01 00:20:43 [Server] INFO [Economy] Essentials Economy found: Waiting
26.01 00:20:43 [Server] INFO [Permission] SuperPermissions loaded as backup permission system.
26.01 00:20:43 [Server] INFO Enabled Version 1.7.3-b131
26.01 00:20:43 [Server] INFO Registered Vault permission & chat hook.
26.01 00:20:43 [Server] INFO Enabling LiteEco v1.2.3-SNAPSHOT
26.01 00:20:43 [Server] INFO PlaceholderAPI hook not found
26.01 00:20:43 [Server] INFO Registered Vault like a service.
26.01 00:20:43 [Server] INFO Treasury not found, for better experience please download Treasury or Vault.
26.01 00:20:43 [Server] INFO You are using current version !
26.01 00:20:43 [Server] INFO Registering commands with Cloud Command Framework !
26.01 00:20:43 [Server] INFO Bukkit Listener AccountEconomyManageListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener PlayerEconomyPayListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener AdminEconomyGlobalDepositListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener AdminEconomyGlobalSetListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener AdminEconomyGlobalWithdrawListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener AdminEconomyMoneyDepositListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener AdminEconomyMoneyWithdrawListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener AdminEconomyMoneySetListener registered () -> ok
26.01 00:20:43 [Server] INFO Bukkit Listener PlayerJoinListener registered () -> ok
26.01 00:20:43 [Server] INFO Listeners registered(9) in time 21 ms -> ok
26.01 00:20:43 [Server] INFO Plugin enabled in time 512 ms
26.01 00:20:43 [Server] INFO Enabling WorldEdit v7.2.13+46576cc
26.01 00:20:43 [Server] INFO Registering commands with com.sk89q.worldedit.bukkit.BukkitServerInterface
26.01 00:20:43 [Server] INFO WEPIF: Vault detected! Using Vault for permissions
26.01 00:20:44 [Server] INFO Using com.sk89q.worldedit.bukkit.adapter.impl.v1_19_R2.PaperweightAdapter as the Bukkit adapter
26.01 00:20:46 [Server] INFO Preparing level "world"
26.01 00:20:47 [Server] INFO Preparing start region for dimension minecraft:overworld
26.01 00:20:47 [Server] INFO Time elapsed: 185 ms
26.01 00:20:47 [Server] INFO Preparing start region for dimension minecraft:the_nether
26.01 00:20:47 [Server] INFO Time elapsed: 47 ms
26.01 00:20:47 [Server] INFO Preparing start region for dimension minecraft:the_end
26.01 00:20:47 [Server] INFO Time elapsed: 38 ms
26.01 00:20:47 [Server] INFO Enabling PlaceholderAPI v2.11.2
26.01 00:20:47 [Server] WARN Loaded class com.viaversion.viaversion.api.type.Type from ViaVersion v4.5.1 which is not a depend or softdepend of this plugin.
26.01 00:20:48 [Server] INFO Fetching available expansion information...
26.01 00:20:48 [Server] INFO Enabling HolographicDisplays v3.0.1-SNAPSHOT-b244
26.01 00:20:48 [Server] INFO Enabling Votifier v2.7.3
26.01 00:20:48 [Server] INFO Loaded token for website: default
26.01 00:20:48 [Server] INFO Using epoll transport to accept votes.
26.01 00:20:48 [Server] INFO Method none selected for vote forwarding: Votes will not be received from a forwarder.
26.01 00:20:48 [Server] INFO Enabling CommandAPI v8.7.1
26.01 00:20:48 [Server] INFO Chat preview is not available
26.01 00:20:48 [Server] INFO Enabling ViaVersion v4.5.1
26.01 00:20:48 [Server] INFO Votifier enabled on socket /51.81.248.137:8448.
26.01 00:20:48 [Server] INFO Enabling floodgate v2.2.0-SNAPSHOT (b74-4f36112)
26.01 00:20:48 [Server] INFO Enabling ViaBackwards v4.5.1
26.01 00:20:48 [Server] INFO Enabling Geyser-Spigot v2.1.0-SNAPSHOT
26.01 00:20:49 [Server] INFO Enabling Essentials v2.20.0-dev+39-312d169
26.01 00:20:49 [Server] INFO Attempting to convert old kits in config.yml to new kits.yml
26.01 00:20:49 [Server] INFO No kits found to migrate.
26.01 00:20:49 [Server] INFO Loaded 38132 items from items.json.
26.01 00:20:49 [Server] INFO Using locale en_US
26.01 00:20:49 [Server] INFO ServerListPingEvent: Spigot iterator API
26.01 00:20:49 [Server] INFO Starting Metrics. Opt-out using the global bStats config.
26.01 00:20:49 [Server] INFO [Economy] Essentials Economy hooked.
26.01 00:20:49 [Server] INFO Using Vault based permissions (LuckPerms)
26.01 00:20:49 [Server] INFO Enabling MobArena v0.107
26.01 00:20:49 [Server] INFO Vault found; economy rewards enabled.
26.01 00:20:50 [Server] INFO Loaded arena 'default'
26.01 00:20:50 [Server] INFO Loaded arena 'Pyramid'
26.01 00:20:50 [Server] INFO Loaded 3 sign templates.
26.01 00:20:50 [Server] INFO Enabling WorldGuard v7.0.8-beta-01+cbb2ba7
26.01 00:20:50 [Server] INFO (world) TNT ignition is PERMITTED.
26.01 00:20:50 [Server] INFO (world) Lighters are PERMITTED.
26.01 00:20:50 [Server] INFO (world) Lava fire is PERMITTED.
26.01 00:20:50 [Server] INFO (world) Fire spread is UNRESTRICTED.
26.01 00:20:50 [Server] INFO Loaded configuration for world 'world'
26.01 00:20:50 [Server] INFO (world_nether) TNT ignition is PERMITTED.
26.01 00:20:50 [Server] INFO (world_nether) Lighters are PERMITTED.
26.01 00:20:50 [Server] INFO (world_nether) Lava fire is PERMITTED.
26.01 00:20:50 [Server] INFO (world_nether) Fire spread is UNRESTRICTED.
26.01 00:20:50 [Server] INFO Loaded configuration for world 'world_nether'
26.01 00:20:50 [Server] INFO (world_the_end) TNT ignition is PERMITTED.
26.01 00:20:50 [Server] INFO (world_the_end) Lighters are PERMITTED.
26.01 00:20:50 [Server] INFO (world_the_end) Lava fire is PERMITTED.
26.01 00:20:50 [Server] INFO (world_the_end) Fire spread is UNRESTRICTED.
26.01 00:20:50 [Server] INFO Loaded configuration for world 'world_the_end'
26.01 00:20:50 [Server] INFO Loading region data...
26.01 00:20:50 [Server] INFO Enabling AdvancedPortals v0.9.2
26.01 00:20:50 [Server] INFO BLOCK_PORTAL_TRAVEL found
26.01 00:20:50 [Server] WARN Proxy features disabled for Advanced Portals as bungee isn't enabled on the server (spigot.yml) or if you are using Paper settings.velocity-support.enabled may not be enabled (paper.yml)
26.01 00:20:50 [Server] INFO Advanced portals have been successfully enabled!
26.01 00:20:50 [Server] INFO Enabling Spartan vPhase 490.1
26.01 00:20:51 [Server] INFO Enabling Citizens v2.0.30-SNAPSHOT (build 2890)
26.01 00:20:51 [Server] INFO Loading external libraries
26.01 00:20:51 [Server] INFO Successfully registered expansion: citizensplaceholder [1.0.0]
26.01 00:20:51 [Server] INFO Loaded economy handling via Vault.
26.01 00:20:51 [Server] INFO Enabling DiscordSRV v1.26.0
26.01 00:20:52 [Server] INFO Enabling TradeSystem v2.1.3
26.01 00:20:52 [Server] INFO __________________________________________________________
26.01 00:20:52 [Server] INFO TradeSystem [2.1.3]
26.01 00:20:52 [Server] INFO Status:
26.01 00:20:52 [Server] INFO MC-Version: 1.19.3 (R0.1-SNAPSHOT, Paper)
26.01 00:20:52 [Server] INFO > Loading sounds
26.01 00:20:52 [Server] INFO > Loading blacklist
26.01 00:20:52 [Server] INFO ...got 3 blocked item(s)
26.01 00:20:52 [Server] INFO > Loading layouts
26.01 00:20:52 [Server] INFO ...got 4 layout(s)
26.01 00:20:52 [Server] INFO > Database logging is disabled
26.01 00:20:52 [Server] INFO Finished (277ms)
26.01 00:20:52 [Server] INFO __________________________________________________________
26.01 00:20:52 [Server] INFO Successfully registered expansion: tradesystem [1.1]
26.01 00:20:52 [Server] INFO Enabling CombatLogX v11.1.0.7.1025
26.01 00:20:52 [Server] INFO CombatLogX was loaded successfully.
26.01 00:20:52 [Server] INFO Enabling expansions...
26.01 00:20:52 [Server] INFO Enabling expansion 'EssentialsX Compatibility v16.3'...
26.01 00:20:52 [Server] INFO [EssentialsX Compatibility] Successfully found a dependency: Essentials v2.20.0-dev+39-312d169
26.01 00:20:52 [Server] INFO Enabling expansion 'PlaceholderAPI Compatibility v16.9'...
26.01 00:20:52 [Server] INFO [PlaceholderAPI Compatibility] Successfully found a dependency: PlaceholderAPI v2.11.2
26.01 00:20:52 [Server] INFO Successfully registered expansion: combatlogx [16.9]
26.01 00:20:52 [Server] INFO Enabling expansion 'WorldGuard Compatibility v16.4'...
26.01 00:20:52 [Server] INFO [WorldGuard Compatibility] Successfully found a dependency: WorldGuard v7.0.8-beta-01+cbb2ba7
26.01 00:20:52 [Server] INFO Enabling expansion 'Scoreboard v16.12'...
26.01 00:20:52 [Server] INFO Successfully enabled 4 expansions.
26.01 00:20:52 [Server] INFO CombatLogX was enabled successfully.
26.01 00:20:52 [Server] INFO Enabling EssentialsSpawn v2.20.0-dev+39-312d169
26.01 00:20:52 [Server] INFO Starting Metrics. Opt-out using the global bStats config.
26.01 00:20:52 [Server] INFO Enabling CoreProtect v21.3
26.01 00:20:52 [Server] INFO CoreProtect has been successfully enabled! 
26.01 00:20:52 [Server] INFO Using SQLite for data storage.
26.01 00:20:52 [Server] INFO --------------------
26.01 00:20:52 [Server] INFO Enjoy CoreProtect? Join our Discord!
26.01 00:20:52 [Server] INFO Discord: www.coreprotect.net/discord/
26.01 00:20:52 [Server] INFO --------------------
26.01 00:20:52 [Server] INFO Enabling Wild v4.4.5-SNAPSHOT
26.01 00:20:52 [Server] INFO [JDA] Login Successful!
26.01 00:20:52 [Server] INFO Enabling dtlTradersPlus v6.4.17
26.01 00:20:52 [Server] INFO dtlTradersPlus> Welcome to dtlTradersPlus v6.4.17!
26.01 00:20:52 [Server] WARN Initializing Legacy Material Support. Unless you have legacy plugins and/or data this is a bug!
26.01 00:20:53 [Server] INFO [JDA] Connected to WebSocket
26.01 00:20:53 [Server] INFO DiscordSRV is up-to-date. (819b02508abe8879cfc2a513fc6cea383b70b5c5)
26.01 00:20:53 [Server] Startup Done loading 11536 rows of data to the Research Engine.
26.01 00:20:53 [Server] INFO [JDA] Finished Loading!
26.01 00:20:53 [Server] INFO Found server G:Nightfall Server(781280721003085844)
26.01 00:20:53 [Server] INFO - TC:👋gates-to-hell(783885798407274556)
26.01 00:20:53 [Server] INFO - TC:announcements(1057100501939933307)
26.01 00:20:53 [Server] INFO - TC:rules(781282562449604628)
26.01 00:20:53 [Server] INFO - TC:❗server-info(1057110323410259968)
26.01 00:20:53 [Server] INFO - TC:✔vote(876850581329821707)
26.01 00:20:53 [Server] INFO - TC:player-info(1063622659683917914)
26.01 00:20:53 [Server] INFO - TC:shops-and-economy(1063948529367777280)
26.01 00:20:53 [Server] INFO - TC:💭suggestions(851588490671423509)
26.01 00:20:53 [Server] INFO - TC:❓-questions(869017742110515210)
26.01 00:20:53 [Server] INFO - TC:❗reports(1057101125158961152)
26.01 00:20:53 [Server] INFO - TC:💬general(875161737899413544)
26.01 00:20:53 [Server] INFO - TC:🎬media(781283309853868033)
26.01 00:20:53 [Server] INFO - TC:👾twitch-streams-and-other-games(865047726307016744)
26.01 00:20:53 [Server] INFO - TC:⚡server-chat(813152744428011580)
26.01 00:20:53 [Server] INFO - TC:💳trading-bounties-services(792207484168699916)
26.01 00:20:53 [Server] INFO - TC:disboard(1065038029439373314)
26.01 00:20:53 [Server] INFO - TC:🗻server-photos(858009473549467678)
26.01 00:20:53 [Server] INFO - TC:⛰old-realm-photos(804518940679733278)
26.01 00:20:53 [Server] INFO - TC:admin-spam(781282754284355604)
26.01 00:20:53 [Server] INFO - TC:🔔aware-of-iron(859887048885403648)
26.01 00:20:53 [Server] INFO - TC:🤖bot-commands(783877273715474433)
26.01 00:20:53 [Server] INFO - TC:admin-spam(781283378057838642)
26.01 00:20:53 [Server] INFO - TC:admin-chat(1057118077029986315)
26.01 00:20:53 [Server] INFO - TC:server-console(783880236441796608)
26.01 00:20:53 [Server] INFO - TC:spartan-detections(1064554251243753632)
26.01 00:20:53 [Server] INFO - TC:invite-logger(924897798363684895)
26.01 00:20:53 [Server] INFO - TC:bots(1056356043950669834)
26.01 00:20:53 [Server] INFO - TC:to-do-list(1057093433119998042)
26.01 00:20:53 [Server] INFO Console forwarding assigned to channel TC:server-console(783880236441796608)
26.01 00:21:01 [Server] INFO dtlTradersPlus> Trying to hook into any Vault supported plugin...
26.01 00:21:01 [Server] INFO dtlTradersPlus> Vault found! Trying to hook...
26.01 00:21:01 [Server] INFO dtlTradersPlus> Found economy plugin: LiteEco
26.01 00:21:01 [Server] INFO dtlTradersPlus> Loaded 3 guis from the guis config!
26.01 00:21:01 [Server] INFO dtlTradersPlus> Hooking into Citizens!
26.01 00:21:01 [Server] INFO dtlTradersPlus> Hooking into PlaceholderAPI
26.01 00:21:01 [Server] INFO dtlTradersPlus> dtlTraders is managed by Degitise. www.degitise.com
26.01 00:21:01 [Server] INFO Enabling Essentials hook
26.01 00:21:01 [Server] INFO Enabling CrateReloaded v2.0.26
26.01 00:21:01 [Server] INFO Config Version: 2.X
26.01 00:21:01 [Server] INFO v1_19_R2
26.01 00:21:01 [Server] INFO Hologram Provider: HolographicDisplays
26.01 00:21:01 [Server] INFO Crate Configurations (1): crate.yml 
26.01 00:21:02 [Server] INFO Enabling VotingPlugin v6.10.0
26.01 00:21:03 [Server] INFO Giving VotingPlugin.Player permission by default, can be disabled in the config
26.01 00:21:03 [Server] INFO Enabled VotingPlugin 6.10.0
26.01 00:21:03 [Server] INFO Enabling LuckPerms hook
26.01 00:21:03 [Server] INFO Enabling PlaceholderAPI hook
26.01 00:21:04 [Server] INFO Placeholder expansion registration initializing...
26.01 00:21:04 [Server] INFO ******************************************
26.01 00:21:04 [Server] INFO Loading Geyser version 2.1.0-SNAPSHOT (git-master-0b80c58)
26.01 00:21:04 [Server] INFO ******************************************
26.01 00:21:07 [Server] INFO Started Geyser on 51.81.248.137:25565
26.01 00:21:07 [Server] Startup Done (3.668s)! Run /geyser help for help!
26.01 00:21:07 [Server] INFO Running delayed init tasks
26.01 00:21:07 [Server] INFO Fetching version information...
26.01 00:21:07 [Server] INFO Linked 2 Bukkit permissions to commands
26.01 00:21:07 [Server] INFO Reloading datapacks...
26.01 00:21:07 [Server] INFO Checking for Updates ... 
26.01 00:21:08 [Server] INFO [Update Checker] A possible update was found for plugin 'BlueSlimeCore'.
26.01 00:21:08 [Server] INFO [Update Checker] Current Version: 2.6.1.222-Beta
26.01 00:21:08 [Server] INFO [Update Checker] New Version: 2.6.0.18
26.01 00:21:08 [Server] INFO [Update Checker] Download Link: https://www.spigotmc.org/resources/83189/
26.01 00:21:08 [Server] INFO [Update Checker] There are no updates available for plugin 'CombatLogX'.
26.01 00:21:08 [Server] INFO Loaded 7 recipes
26.01 00:21:08 [Server] WARN You're 1 EssentialsX dev build(s) out of date!
26.01 00:21:08 [Server] WARN Download it here: https://essentialsx.net/downloads.html
26.01 00:21:08 [Server] INFO No new version available
26.01 00:21:09 [Server] INFO Finished reloading datapacks
26.01 00:21:09 [Server] INFO ViaVersion detected server version: 1.19.3 (761)
26.01 00:21:09 [Server] INFO Essentials found a compatible payment resolution method: Vault Compatibility Layer (v1.7.3-b131)!
26.01 00:21:09 [Server] INFO WorldEdit logging successfully initialized.
26.01 00:21:09 [Server] INFO Successfully registered expansion: discordsrv [1.26.0]
26.01 00:21:09 [Server] WARN Loaded class net.milkbowl.vault.economy.Economy from Vault v1.7.3-b131 which is not a depend or softdepend of this plugin.
26.01 00:21:09 [Server] INFO Successfully registered expansion: vault [1.7.1]
26.01 00:21:09 [Server] INFO 1 placeholder hook(s) registered!
26.01 00:21:09 [Server] INFO Loaded 6 NPCs.
26.01 00:21:09 [Server] INFO Successfully enabled 0 late-load expansions.
26.01 00:21:09 [Server] INFO Finished mapping loading, shutting down loader executor!
26.01 00:21:09 [Server] Startup Done (36.878s)! For help, type "help"
26.01 00:21:09 [Server] INFO Timings Reset
26.01 00:21:09 [Server] INFO Successfully hooked into vault economy!
26.01 00:21:09 [Server] INFO Hooked into vault permissions
```

**Comment by EncryptSL** [comment]
Bug confirmed. and fixed by release 1.2.4-SNAPSHOT.

![image](https://user-images.githubusercontent.com/9441083/214774764-44413fda-df5d-4201-bdc9-5817e2a2b0bd.png)

**Comment by EncryptSL** [comment]
Closed by commit: https://github.com/EncryptSL/LiteEco/commit/394a9bb2343313d060dc052e46071b5a93ca814e
Link to fixed release https://www.spigotmc.org/resources/liteeco-simple-economy-plugin-1-18-x-1-19.101934/update?update=487186
